### PR TITLE
v2: rename package from tailscale -> tsclient

### PR DESCRIPTION
**Pull request by oxtoacart**
Some consumers of this SDK, for example terraform-provider-tailscale, use other packages named 'tailscale'. To avoid having to alias when importing this package, rename to 'tsclient' to avoid most collisions by default.

Updates tailscale/corp#21867

### CODE REVIEW

File: v2/devices.go [hunk 1]
```diff
@@ -143,7 +143,7 @@ func (dr *DevicesResource) SetKey(ctx context.Context, deviceID string, key Devi
 }
 
 // SetDeviceIPv4Address sets the Tailscale IPv4 address of the device.
-func (dr *DevicesResource) SetDeviceIPv4Address(ctx context.Context, deviceID string, ipv4Address string) error {
+func (dr *DevicesResource) SetIPv4Address(ctx context.Context, deviceID string, ipv4Address string) error {
```
Comment (oxtoacart) [inline v2/devices.go:146]: Drive-by: remove stuttering `Device`.

**Review by mpminardi — APPROVED** [review]
